src/scss/CindyJS.scss

```diff
@@ -0,0 +1,69 @@
+// This stylesheet contains styling which is relevant to the
+// functionality of the CindyJS user interface.
+// See http://sass-lang.com/ for SASS language documentation.
+
+@mixin naked {
+    margin: 0px;
+    padding: 0px;
+    border: none;
+}
+
+$button-shadow: inset 1px 1px 2px #fff, inset -1px -1px 2px #aaa;
+$checked-shadow: inset 1px 1px 2px #999, inset -1px -1px 2px #ccc;
+$focused-shadow: 0px 0px 3px #06f;
+
+.CindyJS-widget {
+    overflow: hidden;
+    z-index: 0;
+
+    .CindyJS-button {
+        @include naked;
+        position: absolute;
+        white-space: nowrap;
+        margin-top: -1000px;
+
+        span {
+            @include naked;
+        }
+
+        img {
+            @include naked;
+            display: inline;
+            position: static;
+            vertical-align: baseline;
+            visibility: hidden;
+            margin-right: -1px;
+        }
+
+        input[type=checkbox] {
+            position: absolute;
```

Comment:
I, too, but in my experience a checkbox with `display: none` cannot have the input focus, so it won't work with keyboard navigation using <kbd>⇥</kbd> and <kbd>space</kbd>.